xstac/_xstac.py

```diff
@@ -9,12 +9,39 @@
 import pystac
 import pandas as pd
 from pyproj import CRS, Transformer
-from typing import List, Dict
+from typing import Dict
 
 from ._types import TemporalDimension, HorizontalSpatialDimension, Datacube, Variable
 
 SCHEMA_URI = "https://stac-extensions.github.io/datacube/v2.0.0/schema.json"
 
+CF_STANDARD_AXES = dict(temporal_dimension="T", x_dimension="X", y_dimension="Y")
+
+
+def maybe_use_cf_standard_axis(kw, kw_name, ds):
+    GENERIC_SUGGESTION = (
+        f"Alternatively, pass `{kw_name}` as a string cooresponding to the name of the "
+        f"dataset's {kw_name}."
+        # "If the dataset does not have a {kw_name}, pass `{kw_name}=False`."
+    )
+    if not hasattr(ds, "cf"):
+        raise AttributeError(
+            f"Kwarg `{kw_name}` is None and the dataset does not have a `cf` namespace. Please "
```

Comment:
Is this error message right? I would expect that importing `cf_xarray` here would ensure that the accessor is available, regardless of whether it was imported when the Dataset was created.

Reply:
> Is this error message right?

No. Removed in https://github.com/TomAugspurger/xstac/pull/20/commits/b1b400477a3c4a6b69ca9206fdecf9219e903b71

>  I would expect that importing cf_xarray here would ensure that the accessor is available

Correct. I hadn't realized this. And I think removing the `cf_xarray` import in `conftest.py` (where the test datasets are created) serves as a de-facto test/verification of this expectation: https://github.com/TomAugspurger/xstac/pull/20/commits/b1b400477a3c4a6b69ca9206fdecf9219e903b71#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128